chevah/compat/tests/normal/test_capabilities.py

```diff
@@ -218,17 +251,121 @@ def test_isPrivilegeEnabled_disabled(self):
         privilege = win32security.SE_SECURITY_NAME
         self.assertFalse(self.capabilities._isPrivilegeEnabled(privilege))
 
-    def test_isPrivilegeEnabled_absent(self):
+    def test_symbolic_link(self):
         """
-        Returns False for a privilege which is not present.
+        Not supported on Windows without elevated permissions.
         """
-        # By default SE_RELABEL_NAME should not be available to test
-        # accounts.
-        privilege = win32security.SE_RELABEL_NAME
-        self.assertEqual(
-            u'absent', self.capabilities._getPrivilegeState(privilege))
+        symbolic_link = self.capabilities.symbolic_link
 
-        self.assertFalse(self.capabilities._isPrivilegeEnabled(privilege))
+        self.assertFalse(symbolic_link)
+
+    def test_impersonate_local_account_windows(self):
+        """
+        Impersonation is not available when running as a normal user.
+        """
+        result = self.capabilities.impersonate_local_account
+
+        self.assertFalse(result)
+
+    def test_create_home_folder(self):
+        """
+        On Windows home folders can be created if required privileges
+        are configured for the process.
+        """
+        result = self.capabilities.create_home_folder
+
+        if 'win-xp' in self.hostname:
+            self.assertFalse(result)
+        else:
+            self.assertTrue(result)
+
+    def test_getCurrentPrivilegesDescription(self):
+        """
+        Check getCurrentPrivilegesDescription.
+        """
+        text = self.capabilities.getCurrentPrivilegesDescription()
+
+        self.assertContains('SeChangeNotifyPrivilege:3', text)
+
+        if 'win-xp' in self.hostname:
+            self.assertNotContains('SeCreateSymbolicLinkPrivilege', text)
```

Comment:
we need a comment to explain why we have this branch.

doing assertions inside an if branch is a bad thing to do in testing... usually a smell that the test should be split... but if we don't split it, then we need a comment as a placeholder for the docstring which would have been associated with the splitted test.